app/components/modelling/galaxyScore.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+// grabs the model score (calculated in modelCanvas) and displays it with a
+// progress bar
+const ModelScore = (props) => {
+  if (props.workflow && props.workflow.configuration &&
+    props.workflow.configuration.metadata &&
+    props.workflow.configuration.metadata.modelScore) {
+    let s = props.workflow.configuration.metadata.modelScore;
```

Comment:
This component just renders a score, so pass the score directly as a prop and bypass the mess of conditionals here.